ArduCopter/mode_autotune.cpp

```diff
@@ -487,11 +488,11 @@ void Copter::ModeAutoTune::autotune_attitude_control()
         // hold the copter level for 0.5 seconds before we begin a twitch
         // reset counter if we are no longer level
         if (!currently_level()) {
-            step_start_time = millis();
+            step_stop_time = millis()+AUTOTUNE_REQUIRED_LEVEL_TIME_MS;
```

Comment:
**Nitpicking**: interesting that you chose to use `step_stop_time` for the required level and `step_start_time` for the timeout. I think the opposite was better:

- less code changes
- it seems to me that *stop* relates better to a timeout then _start_; the time at which we will stop is always fixed (due to timeout), but the time it started will be changing if we aren't level

Reply:
I agree and I think I have done a better job in #10081